### ISSUE TITLE
There are different parameters restored by two ways?

### ISSUE THREAD

**Issue by tjusxh** [davidsandberg/facenet#825]
I learned the code from facenet and print the parameters' names by two ways. Method1 prints 492 parameters and method2 prints 490 parameters excludes logits/weights:0 and logits/biases:0. what's the reason?
Method 1:
  with tf.Graph().as_default():
        with tf.Session() as sess:
            model_exp="model/20180402-114759"
            meta_file, ckpt_file = get_model_filenames("model/20180402-114759/")
            saver = tf.train.import_meta_graph(os.path.join(model_exp, meta_file))
            saver.restore(tf.get_default_session(), os.path.join(model_exp, ckpt_file))
            variables = tf.trainable_variables()
            values = sess.run(variables)
            for k,v in zip(variables,values):
                print(k.name)

Method2:
meta_file, ckpt_file = get_model_filenames("model/20180402-114759/")
        sess.run(tf.initialize_all_variables())
        saver.restore(sess, os.path.join("model/20180402-114759/",ckpt_file))
        graph = tf.get_default_graph()
        values = sess.run(var_to_restore)
        for k,v in zip(var_to_restore,values):
             print(v)
